File: Front-End/mock/address.ts
export default {
    'GET /api/address/getAddress':{
        code: "000",
        data: [
            {
                "id":1,
                "userId":1,
                "receiverName":"angle",
                "phoneNumber":"13382839923",
                "addressDetail":"淞沪路2006号",
                "addressProvince":"上海市",
                "addressCity":"上海",
                "addressDistinct":"杨浦区",
            },
            {
                "id":2,
                "userId":1,
                "receiverName":"angle",
                "phoneNumber":"13382839923",
                "addressDetail":"韶山南路",
                "addressProvince":"湖南省",
                "addressCity":"长沙市",
                "addressDistinct":"天心区",
            },
        ],
        msg: "成功获取列表"
    },
}

File: order/src/main/java/fd/adse/order/entity/Address.java
package fd.adse.order.entity;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;

@Entity
@Getter
@Setter
public class Address extends BaseEntity{

    private Long userId;
    private String receiverName;
    private String phoneNumber;
    private String addressProvince;
    private String addressCity;
    private String addressDistinct;
    private String addressDetail;

    public Address() {
    }

    public Address(Long userId, String receiverName, String phoneNumber, String addressProvince, String addressCity, String addressDistinct, String addressDetail) {
        this.userId = userId;
        this.receiverName = receiverName;
        this.phoneNumber = phoneNumber;
        this.addressProvince = addressProvince;
        this.addressCity = addressCity;
        this.addressDistinct = addressDistinct;
        this.addressDetail = addressDetail;
    }

    public String getAddress() {
        return addressProvince + " " + addressCity + " " + addressDistinct + " " + addressDetail;
    }
}
